core/src/mill/define/Applicative.scala

```diff
@@ -32,20 +32,24 @@ object Applicative {
     def ctx()(implicit c: Ctx) = c
     def underlying[A](v: W[A]): T[_]
 
-    def zipMap[R]()(cb: Ctx => Z[R]) = mapCtx(zip()){ (_, ctx) => cb(ctx)}
+    def zipMap[R]()(cb: Ctx => Z[R]) = mapCtx(zip()) { (_, ctx) =>
+      cb(ctx)
+    }
```

Comment:
Honestly I don't think this is worth new-lining; the original is already pretty short & concise as-is, and the two new lines have basically nothing on them